x-pack/legacy/plugins/actions/server/action_type_registry.ts

```diff
@@ -61,6 +61,14 @@ export class ActionTypeRegistry {
       [`actions:${actionType.id}`]: {
         title: actionType.name,
         type: `actions:${actionType.id}`,
+        maxAttempts: actionType.maxAttempts || 1,
+        getRetry(attempts: number, error: any) {
+          if (error instanceof ExecutorError) {
+            return error.retry == null ? true : error.retry;
```

Comment:
Yeah retry by default seems wrong here to me.

Reply:
We would change the logic to never retry unless specified within the `retry` attribute as a Date or `true`?

Reply:
I think so.

Reply:
Not a problem, I have made the changes.